plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployJob.java

```diff
@@ -82,12 +93,21 @@ public IStatus runInWorkspace(IProgressMonitor monitor) throws CoreException {
   }
 
   private CloudSdk getCloudSdk() {
-    MessageConsole messageConsole =
-        MessageConsoleUtilities.getMessageConsole(CONSOLE_NAME, null, true /* show */);
+    MessageConsole messageConsole = MessageConsoleUtilities.getMessageConsole(CONSOLE_NAME, null, true /* show */);
     final MessageConsoleStream outputStream = messageConsole.newMessageStream();
     CloudSdk cloudSdk = new CloudSdk.Builder()
                           .addStdOutLineListener(new MessageConsoleWriterOutputLineListener(outputStream))
                           .addStdErrLineListener(new MessageConsoleWriterOutputLineListener(outputStream))
+                          .appCommandCredentialFile(LoginCredentialExporter.getCredentialFilePath(workDirectory).toFile())
+                          .exitListener(new ProcessExitListener() {
+                            // temporary way of error handling, after #439 is fixed, it'll be cleaner
+                            @Override
+                            public void onExit(int exitCode) {
```

Comment:
does it really need to be indented this much? If so, maybe pull out to an inner class